### PR TITLE
PLANET-7616: Merge CSS core overrides folders

### PR DESCRIPTION
**Pull request by mardelnet**
Ref: https://jira.greenpeace.org/browse/PLANET-7616

<hr>

**DESCRIPTION:**
This PR:
- Moves the files from the [core-overrides](https://github.com/greenpeace/planet4-master-theme/tree/main/assets/src/scss/core-overrides) folder to the [blocks/core-overrides](https://github.com/greenpeace/planet4-master-theme/tree/main/assets/src/scss/blocks/core-overrides).
- Merges the CSS related to the image and the heading blocks.
- Removes duplicated code.
- Removes imports of the files that were part of the `core-overrides` folder.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [rhea](https://www-dev.greenpeace.org/test-rhea) | [admin](https://www-dev.greenpeace.org/test-rhea/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-rhea/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-rhea) | [composer-local.json](https://github.com/greenpeace/planet4-test-rhea/blob/main/composer-local.json)

:watch: 2024.10.24 13:42:18

File: assets/src/scss/blocks/core-overrides/Heading.scss [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-  vertical-align: middle;
+.wp-block-heading {
+  &.is-style-chevron::after {
+    content: "";
+    pointer-events: none;
+    margin-inline-start: $sp-1;
+    height: 12px;
+    width: 12px;
+    display: inline-block;
+    mask-image: url("../../public/images/icons/chevron.svg");
```
Comment (mleray) [inline assets/src/scss/blocks/core-overrides/Heading.scss:9]: You need to update this URL, it no longer works

Comment (mardelnet) [reply assets/src/scss/blocks/core-overrides/Heading.scss]: Done!

**Review by mleray — APPROVED** [review]
Looks good to me 👏

**Pull request merged**